### ISSUE TITLE
OneTrust Cookie Banner and Privacy Notice being blocked by Ghostery enabled browsers

### ISSUE THREAD

**Issue by stefanispublicis** [ghostery/ghostery-extension#1891]
Our OneTrust Cookie Banner and Privacy Notices are being blocked by browsers that have Ghostery enabled and the OneTrust domain needs to be allowed.

**Comment by philipp-classen** [comment]
Can you please share same examples that are affected?

We also have a dedicated repository for breakage. As I understand, this would to best reported there: https://github.com/ghostery/broken-page-reports

**Comment by seia-soto** [comment]
Closing as a duplicate of https://github.com/ghostery/broken-page-reports/issues/1065
